### PR TITLE
Docs folder standardization and other fixes

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,49 +34,53 @@ composer require yiisoft/auth-jwt
...
-            // your signature algorithms
-        ],
-        'serializers' => [
-            // your token serializers
-        ],
-        'key' => [
-            'secret' => 'your-secret',
-            'file' => 'your-certificate-file',
-        ],
+
```
Comment (samdark) [inline README.md:37]: It was good as is. It's meant to be code within point `1.`.

File: README.md [hunk 2]
```diff
@@ -34,49 +34,53 @@ composer require yiisoft/auth-jwt
...
-           '__construct()' => [
-               $params['yiisoft/auth-jwt']['key']['secret']
-           ],
-       ],
-       
-       AuthenticationMethodInterface::class => JwtMethod::class,
-   ];
-   ```
-   > Note: Don't forget to declare your implementations of `\Yiisoft\Auth\IdentityInterface` and `\Yiisoft\Auth\IdentityRepositoryInterface`.
+
```
Comment (samdark) [inline README.md:55]: It was good as is.

**Review by samdark — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (samdark) [reply README.md:55]: There should be no blank line.

**Review by samdark — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (luizcmarin) [reply README.md:55]: markdownlint plugin is to blame.

File: LICENSE.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
-Copyright © 2008 by Yii Software (https://www.yiiframework.com/)
+Copyright © 2008 by Yii Software (<https://www.yiiframework.com/>)
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without
 modification, are permitted provided that the following conditions
 are met:
 
- * Redistributions of source code must retain the above copyright
+* Redistributions of source code must retain the above copyright
```
Comment (vjik) [inline LICENSE.md:8]: Align it vertically, like here: https://raw.githubusercontent.com/yiisoft/package-template/master/LICENSE.md

File: README.md [hunk 4]
```diff
@@ -97,30 +96,26 @@ $middleware = new \Yiisoft\Auth\Middleware\Authentication(
...
 
-The package is tested with [PHPUnit](https://phpunit.de/). To run tests:
+- [Internals](docs/internals.md)
 
-```shell
-./vendor/bin/phpunit
-```
-
-## Mutation testing
+## Support
```
Comment (vjik) [inline README.md:104]:
```suggestion
```

**Review by vjik — CHANGES_REQUESTED** [review]

**Review by vjik — APPROVED** [review]

**Pull request merged**